### PR TITLE
Login task

### PR DESCRIPTION
**Pull request by MarlenaKarpiej**
Created registration and login for users

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,4 @@ build/
 
 ### VS Code ###
 .vscode/
+src/main/resources/application-local.properties
```
Comment (pbielicki) [inline .gitignore:32]: why is it ignored?

File: pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.2.5.RELEASE</version>
+		<version>2.1.5.RELEASE</version>
```
Comment (pbielicki) [inline pom.xml:8]: why downgrade?

**Review by pbielicki — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (MarlenaKarpiej) [reply .gitignore:32]: is ignored because my apyKey is saved there

File: pom.xml (hunk 2 above)
Comment (MarlenaKarpiej) [reply pom.xml:8]: I changed the version because was a problem to start the application.

**Pull request merged**